scopa.py

```diff
@@ -0,0 +1,217 @@
+import random
+import tactics
+
+# all possible colors: Denari, Bastoni, Spade, Coppe
+colors = [tactics.denaro, " ", " ", " "]
+hand_size = 3
+initial_table_size = 4
+no_of_players = 2
+no_take = -99
+
+
+class Scopa:
+    # card deck
+    deck = []
+    # all hands
+    hands = []
+    # all piles
+    piles = []
+    # current table
+    table = []
+    # how many scopas
+    scopa_count = []
+    # how many points
+    points = []
+    # what was the last hand played
+    last_hand_played = -1
+
+    def __init__(self):
+        self.generate_deck()
+        # self.deck = ['03 ', '06 ', '05 ', '12 ', '01 ', '12*', '05*', '03 ', '02 ', '03 ', '11*', '05 ', '04 ', '03*', '01 ', '04*', '06*', '04 ', '02 ', '07 ', '13 ', '13*', '01*', '01 ', '07 ', '07*', '11 ', '07 ', '02 ', '12 ', '13 ', '06 ', '12 ', '11 ', '06 ', '04 ', '13 ', '02*', '05 ', '11 ']
+        self.initiate_play()
+
+    # this method generates random deck
+    def generate_deck(self):
```

Comment:
Method name could be like `generate_random_deck` and then comment would not be needed